site/layouts/shortcodes/front_matter_img.html

```diff
@@ -0,0 +1,47 @@
+{{/* this code is here incase we want to refer to an image in the front matter and have it be responsive */}}
```

Comment:
shortcodes won't work in a layout file so I think this one needs to go into a partial instead if you want to be able to use it from a layout.